seqr/views/apis/individual_api_2_3_tests.py

```diff
@@ -202,12 +204,12 @@ def test_individuals_table_handler(self):
 "1"	"NA19678"	""	""	""	"Male"	"Unaffected"	"a individual note"	""\n\
 "21"	"HG00735"	""	""	""	"Female"	"Unaffected"	""	"a new family"'
 
-        f = SimpleUploadedFile("1000_genomes demo_individuals.tsv", data)
+        f = SimpleUploadedFile("1000_genomes demo_individuals.tsv", data.encode('utf-8'))
```

Comment:
Why does the upload file content type need to change? We need to continue to support all the same file formats as before so it is concerning to me that the test input needs to change now. Theres no reason to believe the real-world tsvs are going to be utf-8 encoded...

Reply:
`SimpleUploadedFile(filename, data)` requires the data must be a `bytes-like` type. It can't accept `str` type data. In Python 2, data can be an `str` instance because Python 2 `str` IS a `bytes-like` type. But In Python 3, `str` is completely different from `bytes`, it is actually `unicode` in Python 2.
The real-world `.tsv` files can have any encoding but they must be decoded each time when they are read into a Python 3 `str` instance. The encoding here does nothing with the encoding of `.tsv` files.